data/external/scrapers/public_transport.py

```diff
@@ -0,0 +1,66 @@
+from pathlib import Path
+from external.scraping_utils import cached_json
+from external.scrapers.roomfinder import scrape_maps
+import csv
+from decimal import Decimal
+from geopy.distance import distance
+
+# CSV indexes
+STATIONID = 0
+NAME = 1
+ORT = 2
+GLOBAL_ID = 3
+WGS84X = 4  # lat
+WSG84Y = 6  # lon
+
+MAXDISTANCE = 1  # max distance from building
+
+
+def avg(x1, x2):
+    return (x1 + x2) / 2
+
+
+def nearby(building_coords:tuple, stations: list[dict]) -> list[dict]:
+    results=[]
+    for station in stations:
+        distance_to_building=round(distance((station.get("lat"), station.get("lon")), (building_coords)).km,2)
```

Comment:
Rounding here might be excessively strict and deprive the frontend from differentiating between close stations
=> Is there is a reasoning why rounding is better?

Reply:
The main benefit here is for better readability. I thought it would be best to see the distance in km with 2 decimal places. 
I am not sure what you mean with differentiating issues. each station is stored like this: 
```
      { "distance": 0.62,
        "global-id": "",
        "id": "8007",
        "lat": "48.1472252045499",
        "lon": "11.5605495171959",
        "name": "Brienner Str.",
        "ort": "M\u00fcnchen" 
      }
```
While rounding would make it impossible to sort by distance with 100% accuracy, I think there is enough information to differentiate stations.

Reply:
Thinking about this a bit more:
Storing `0.xx km` here does not super make sense imo, I would prefer to display `xxx meters away` in the frontend, as that is probably more user-friendly than listing a bunch of `0.xxx km` ^^

Reply:
nearby now returns meters